### PR TITLE
updated

### PR DESCRIPTION
**Pull request by Rishita-Gusain**
Targets #2980

### Description
removed the padding of the recommend classs

### Checklist:p! -->
- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by Ayush8923** [comment]
@Rishita-Gusain Please assign yourself and also assign the reviewers for review. And put the label `ready-for-review`.

File: resources/views/knowledgecafe/library/books/show_nudge_modal.blade.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <div class="modal-content">
             <div class="modal-header border-bottom-0">
                 <h6 class="modal-title">
-                    <span>You have read</span>    
+                    <span>You have read </span>
```
Comment (Ayush8923) [inline resources/views/knowledgecafe/library/books/show_nudge_modal.blade.php:12]:
```suggestion
                    <span>You have read</span>    
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
When I change the screen size then these 3 filters are not aligned. Please fix this.
Please refer Screenshot below
image

File: resources/js/components/Dashboard/UserDashboardLibrary.vue [hunk 2]
```diff
@@ -8,24 +8,22 @@
...
         </li>
         <li class="nav-item">
-          <a id="read" class="nav-link c-pointer" @click="setActiveTile('read')"
+          <a id="read" class="nav-link c-pointer h-75 d-inline-block" @click="setActiveTile('read')"
             >You have read</a
-          >
+            >
         </li>
 
         <li class="nav-item">
```
Comment (Ayush8923) [inline resources/js/components/Dashboard/UserDashboardLibrary.vue:22]: We need to just add the `d-flex` class. Please test locally once. And also please add the screenshot that's how it looks.

**Comment by Hemrajbharti123** [comment]
**Local Testing Feedback**
Still, the 3 filters are not aligned. Please fix this.



image

**Review by Hemrajbharti123 — APPROVED** [review]

**Comment by Ayush8923** [comment]
Suggested changes have not been implemented and @Rishita-Gusain is not working in the PR. Closing it.

**Pull request closed without merging**